src/form/Select/utils/keyboard.ts

```diff
@@ -0,0 +1,102 @@
+export const keyNameToCode = {
```

Comment:
I've added this mapping to keep readability for 
```
selectOnKeys={['Enter', 'Space', 'Comma']
```
I think this more friendly format instead array of numbers (key codes)

Reply:
You don't really need this with https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

Reply:
This is works for Keyboard events (manually filling) but didn't work on paste due to ClipboardEvent (another way to interact with input needs a little bit different handler)